src/Sensors/Baro/BMP390.cpp

```diff
@@ -3,46 +3,37 @@
 namespace mmfs
 {
 
-    BMP390::BMP390()
+    BMP390::BMP390(const char *name) : bmp()
     {
-        setName("BMP390");
+        setName(name);
     }
 
-    bool BMP390::initialize()
+    bool BMP390::init()
     {
         if (!bmp.begin_I2C())
         { // hardware I2C mode, can pass in address & alt Wire
             // Serial.println("Could not find a valid BMP390 sensor, check wiring!");
             return initialized = false;
         }
 
-        delay(1000);
+        // delay(1000);
```

Comment:
Same question as BMP280 about the delay

Reply:
Most likely to help the sensor stabilize, but by the time this function is run, the board will have been powered for multiple seconds.